SwissKnife/avro/AvroTransformer.py

```diff
@@ -0,0 +1,271 @@
+        cast_dict = {}
+        for field in schema["fields"]:
+            field_name = field["name"]
+            types_list = field["type"]
+            cast_dict[field_name] = types_list
+        return cast_dict
+
+    @staticmethod
+    def _create_defaults_dict(schema: dict) -> dict:
+        """Creates a dict that maps a field with a defaults value.
+        :param schema: The provided avro schema
+        :type schema: dict
+        :return: A dict that maps field with its defaults value.
+        :rtype: dict
+        """
+
+        defaults_dict = {}
+        for field in schema["fields"]:
+            field_name = field["name"]
+            if "default" in field:
+                defaults_value = field["default"]
+                defaults_dict[field_name] = defaults_value
+            else:
+                defaults_dict[field_name] = NoDefault
+        return defaults_dict
+
+    @staticmethod
+    def _get_transform_function(transform_name: str) -> Callable[[object, dict], object]:
+        """Gets a transform function from a transform_name. This purpose of this function
+        is transform the current value of a field to another one (for example, an available
+        value for the field type).
+        transform_name can be directly the name of a function or it can be
+        a expresion, so the result will be a closure. For example,
+        "copyFrom(date)" refers to function "copyFrom" whith the parameter "date".
+        The returned function has two parameters:
+            - value: The current value of the field.
+            - record: The entire record.
+
+        Available functions:
+            - int2boolean: Transform a int to a bolean. True if x > 0, otherwise False.
+            - copyFrom(y): Return the value of field Y. So, the current value of the
+                           target field is ignored.
+
+        :param transform_name: The transform name.
+        :type transform_name: str
+        :raises RuntimeError: Raises Exception if transform_name is invalid.
+        :raises RuntimeError: copyFrom function raises Exception when it is executed
+                              if the provided field is invalid.
+        :return: A transform function.
+        :rtype: Callable[[object, dict], object]
+        """
+        selected_function = None
+        if transform_name == "int2boolean":
+            def int2boolean(value: int, record: dict) -> bool:
+                if value is None:
+                    return None
+                else:
+                    return int(value) > 0
+            selected_function = int2boolean
+        elif re.match("copyFrom\(\w+\)", transform_name):
+            copied_field = re.match("copyFrom\((\w+)\)", transform_name)[1]
+
+            def copyFrom(value:int, record:dict) -> bool:
+                if copied_field in record:
+                    return record[copied_field]
+                else:
+                    raise RuntimeError("Invalid field in copyFrom")
```

Comment:
Maybe the exception message could contain more info to help debug the issue, such as the value of `copied_field` or the available fields in the record